Project cannot be built

```
$ mvn clean install
```

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-resources-plugin:2.7:resources (default-resources) on project wscdn-client: Unable to parse configuration of mojo org.apache.maven.plugins:maven-resources-plugin:2.7:resources for parameter resource: Cannot find default setter in class org.apache.maven.plugin.resources.Resource -> [Help 1]

--- a/pom.xml
+++ b/pom.xml
@@ -123,16 +123,6 @@
             <resource>
                 <directory>src/main/resources</directory>
                 <filtering>true</filtering>
-                <includes>
-                    <include>service.properties</include>
-                </includes>
-            </resource>
-            <resource>
-                <directory>src/main/resources</directory>
-                <filtering>false</filtering>
-                <excludes>
-                    <exclude>service.properties</exclude>
-                </excludes>
             </resource>
         </resources>
 
@@ -150,11 +140,6 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-resources-plugin</artifactId>
                 <version>2.7</version>
-                <configuration>
-                    <resources>
-                        <resource>*</resource> 
-                    </resources>
-                </configuration>
             </plugin>
 
             <!-- Generate javadoc and sources -->

